app/services/invite_provider_user.rb

```diff
@@ -35,6 +35,7 @@ def invite_user_to_dfe_sign_in
       family_name: @provider_user.last_name,
       email: @provider_user.email_address,
       userRedirect: Rails.application.routes.url_helpers.provider_interface_url,
+      inviteBodyOverride: "<p>You've been invited to start managing teacher training applications made through GOV.UK.</p><p>You need to create a DfE Sign-in account to do this.",
```

Comment:
I think the up to date content is in the comment in the document, otherwise this looks good 👍